### PR TITLE
Added Europa Universalis II

### PR DESCRIPTION
**Pull request by ImperatorS79**

### CODE REVIEW

File: Applications/Games/Europa Universalis II/application.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"name":	"Europa Universalis II",
+	"description": "Europa Universalis II is a strategy computer game developed by Paradox Development Studio and published by Strategy First, based on world history spanning a timeline between 1419 through 1820"
```
Comment (plata) [inline Applications/Games/Europa Universalis II/application.json]: Is there a "." missing in the end?

File: Applications/Games/Europa Universalis II/Local/script.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+include(["Engines", "Wine", "QuickScript", "LocalInstallerScript"]);
+
+new LocalInstallerScript()
+    .name("Europa Universalis II")                       
+    .editor("Ubisoft")                               
+    .author("ImperatorS79") 
+    .category("Games")                           
+    .executable("eu2.exe")
+    //TO DO : find static link to 1.07 and 1.09 update, and add a .postinstall.
```
Comment (plata) [inline Applications/Games/Europa Universalis II/Local/script.js]: I guess this works without these updates?

Comment (ImperatorS79) [inline Applications/Games/Europa Universalis II/Local/script.js]: Yes, it does

Comment (ImperatorS79) [inline Applications/Games/Europa Universalis II/Local/script.js]: I would like to provide those updates, but can find a link that stay the same every time, if you see what I try to say...

File: Applications/Games/Europa Universalis II/application.json (hunk 1 above)
Comment (ImperatorS79) [inline Applications/Games/Europa Universalis II/application.json]: Done

File: Applications/Games/Europa Universalis II/Local/script.js (hunk 2 above)
Comment (plata) [inline Applications/Games/Europa Universalis II/Local/script.js]: Yes. It's fine.

**Pull request merged**